source/introduction.txt

```diff
@@ -0,0 +1,263 @@
+  This lets you to avoid fetching unnecessary data.
+
+- **Minimize Roundtrips**: A single GraphQL endpoint handles all requests. 
+  You can group your GraphQL queries into a single request to one endpoint instead of
+  sending requests to multiple endpoints.
+
+{+backend+} generates a GraphQL API for your application based on your :ref:`data model <intro-web-schema>`. 
+The API includes a set of CRUD operations that you can access with a GraphQL client
+or in a HTTP request. You can also create custom resolvers backed by :ref:`functions <intro-web-functions>`.
+
+To learn more about the Realm GraphQL API, see :ref:`graphql-api`.
+
+.. _intro-web-mql:
+
+Client-Side MongoDB Queries
+```````````````````````````
+
+Traditionally developers query the database on the server and access the server
+from the client through a HTTP API. This requires more developer resources and lacks
+the flexibility of directly querying a database.
+
+{+backend+} includes the :manual:`Query API </tutorial/query-documents>`,
+a powerful API that lets you access {+atlas+} data from the client.
+You can query {+atlas-short+} like you're using a database driver and {+service-short+} enforces 
+:ref:`data access rules <intro-web-schema>` rules for all requests.
+
+To learn more about the built-in MongoDB service, see :ref:`data-sources`.
+
+.. _intro-device-sync:
+
+Synchronize Data Across Devices
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Synchronizing data is a hard problem for mobile app developers. Realm Sync
+provides offline-first sync capabilities for your mobile application. 
+Client applications use :ref:`Realm Database <intro-realm-database>` to 
+persist data to the local device. When the device has a network connection, 
+Sync seamlessly sends the data to the Realm App. The app stores the data 
+to a linked :ref:`MongoDB Atlas data source <data-sources>`, and updates 
+the data on other client devices.
+
+Realm Sync provides :ref:`conflict resolution <conflict-resolution>` to 
+handle multiple users updating the same data. When combined with :ref:`user 
+authentication and management <intro-user-authentication>` and :ref:`schema 
+validation and data access <intro-web-schema>`, Realm Sync provides a secure
+way for your mobile clients to sync data across devices and with the cloud.
+
+Get Started with MongoDB Realm
+------------------------------
+
+You can ship your first app built with MongoDB Realm in minutes.
+
+Tutorials
+~~~~~~~~~
+
+Check out our :doc:`tutorials </tutorials>` to learn more about how to build 
+Realm Applications. Tutorials start with git repositories you can clone
+and run, and walk you through code excerpts that are specific to working
+with Realm.
+
+To get started with your first web app, follow the :ref:`Task Tracker 
+application tutorial <tutorial-task-tracker-web>`. You can also refer to the 
+:ref:`Web SDK <web-intro>` documentation for TypeScript and JavaScript.
```

Comment:
could this be generalized for all Realm client SDKs?

Reply:
Now that the sites are separated, and with the plan to deprecate the existing task tracker tutorials, I didn't want to link to them here. Instead, I _just_ wanted to link to the stuff that's relevant to App Services + the template apps I linked to in the next section.

Reply:
it just reads weird to me to refer only to the web sdk w/o mention of the others here.